### PR TITLE
Add configs related to forced challenge questions during login flow

### PR DESCRIPTION
**Pull request by AnuradhaSK**
### Proposed changes in this pull request
Configs related to newly property added in https://github.com/wso2-extensions/identity-governance/pull/383 and `"Enable forced challenge questions"`

### CODE REVIEW

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml [hunk 1]
```diff
@@ -784,11 +784,15 @@
                     <Enable>true</Enable>
                     <MaxFailedAttempts>2</MaxFailedAttempts>
                 </ReCaptcha>
+                <Forced>
```
Comment (tharindu-bandara) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml:787]: Can we move this also under the `<Question>` tag?

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml [hunk 2]
```diff
@@ -784,11 +784,15 @@
...
+                    <Enable>false</Enable>
+                </Forced>
             </Password>
             <Answer>
                 <Regex>.*</Regex>
                 <Uniqueness>false</Uniqueness>
             </Answer>
+            <MinQuestionsToAnswer></MinQuestionsToAnswer>
```
Comment (tharindu-bandara) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml:795]: Better to have a default value. Ex: there is no limit = 0. 

Also, let's add a description to this property. Explaining what it is, and special cases(ex: 0)

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2 [hunk 3]
```diff
@@ -961,11 +961,15 @@
...
                 </ReCaptcha>
+                <Forced>
+                    <Enable>{{identity_mgt.forced_challenge_questions.enable_forced_challenge_questions}}</Enable>
+                </Forced>
             </Password>
             <Answer>
                 <Regex>{{identity_mgt.password_reset_challenge_questions.answer_regex}}</Regex>
                 <Uniqueness>{{identity_mgt.password_reset_challenge_questions.enforce_answer_uniqueness}}</Uniqueness>
             </Answer>
+            <MinQuestionsToAnswer>{{identity_mgt.challenge_questions.min_required_answers}}</MinQuestionsToAnswer>
```
Comment (tharindu-bandara) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2:972]: Please follow the same comments mentioned for the `identity.xml` file.

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json [hunk 4]
```diff
@@ -265,6 +265,9 @@
   "identity_mgt.recovery.challenge_question.enforce_answer_uniqueness": false,
   "identity_mgt.password_reset_challenge_questions.question_separator": "!",
 
+  "identity_mgt.forced_challenge_questions.enable_forced_challenge_questions": false,
+  "identity_mgt.challenge_questions.min_required_answers": "",
```
Comment (tharindu-bandara) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json:269]: Let's use a default value as mentioned earlier.

Comment (AnuradhaSK) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json]: Fixed in 70f7312

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2 (hunk 3 above)
Comment (AnuradhaSK) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2:977]: Fixed in 70f7312

**Pull request merged**